pyproject.toml

```diff
@@ -1,2 +1,12 @@
 [tool.black]
-line-length = 100
\ No newline at end of file
+line-length = 100
+
+[tool.isort]
+line_length = 100
```

Comment:
We can do `profile = "black"` now, see https://pycqa.github.io/isort/docs/configuration/black_compatibility.html (I just switched the server to this: https://github.com/freedomofpress/securedrop/commit/bf09f6135a513f59a9dc434bafae1c232c02b440